src/docserv/bih.py

```diff
@@ -330,11 +330,16 @@ def read_conf_dir(self):
             ('productconfig_simplified_%s.xml' % target))
         logger.debug("Stitching XML config directory to %s",
                      self.stitch_tmp_file)
-        cmd = '%s --simplify --revalidate-only --valid-languages="%s" %s %s' % (
+        cmd = ('%s --simplify --revalidate-only '
+               '--valid-languages="%s" %s %s '
+               '--site-sections="%s"'
+               ) % (
             os.path.join(BIN_DIR, 'docserv-stitch'),
             self.config['server']['valid_languages'],
             self.config['targets'][target]['config_dir'],
-            self.stitch_tmp_file)
+            self.stitch_tmp_file,
+            self.config['targets'][target]['site_sections'],
+            )
```

Comment:
Looks good, but do note that docserv.py has a duplicate of this code (should not be the case but I was/am young/dumb).